pytorch_blade/src/compiler/jit/torch/shape_analysis.cpp

```diff
@@ -2303,8 +2303,20 @@ class ShapePropagator : public PropertyPropBase {
         }
       }
       return success;
-    } else if (node->matches(
-                   "aten::pow(Tensor self, Scalar exponent) -> Tensor")) {
+    } else if (node->matches("aten::bmm(Tensor self, Tensor mat2) -> Tensor")) {
+      auto t1 = tensor_types.at(0);
+      auto t2 = tensor_types.at(1);
+      if (t1 && t2) {
```

Comment:
Nit: emit a warning if the rank of t1 and t2 are not equals to 3?

Reply:
pass the shape inference for op with `rank != 3`